website/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM prestashop/prestashop:1.7.8-apache
+
+COPY ./prestashop.crt /etc/ssl/certs/prestashop.crt
+COPY ./prestashop.crt /usr/local/share/ca-certificates/prestashop.crt
+COPY ./prestashop.key /etc/ssl/private/prestashop.key
+COPY ./prestashop-ssl.conf /etc/apache2/sites-available/prestashop-ssl.conf
+
+RUN update-ca-certificates
+RUN a2enmod ssl
+RUN a2ensite prestashop-ssl
+
```

Comment:
Docker works in such a way, that for each run it creates new layer. Change it to 
RUN update-ca-certificates && a2enmod ssl && a2ensite prestashop-ssl